lcli/src/transition_blocks.rs

```diff
@@ -58,31 +68,58 @@ pub fn run_transition_blocks<T: EthSpec>(
 fn do_transition<T: EthSpec>(
     mut pre_state: BeaconState<T>,
     block: SignedBeaconBlock<T>,
+    signature_strategy: BlockSignatureStrategy,
     spec: &ChainSpec,
 ) -> Result<BeaconState<T>, String> {
+    let t = Instant::now();
     pre_state
         .build_all_caches(spec)
         .map_err(|e| format!("Unable to build caches: {:?}", e))?;
+    println!("Build caches: {}ms", t.elapsed().as_millis());
+
+    let t = Instant::now();
+    pre_state
+        .update_tree_hash_cache()
```

Comment:
I've added a few calls to build caches in order to better capture the typical flow a block undergoes on a live beacon node. Usually we start with states that have their caches fully built, then apply slot processing in the state advance timer, then apply the block and calculate the tree hash root.